scylla/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "scylla"
-version = "0.4.4"
+version = "0.5.0"
```

Comment:
That bump will be done once we're ready to release, via our release script. Scylla is still in 0.x, so we have relaxed policy about specific version, and it's more likely we'll only release 0.5.0 after some new features arrive. Let's just drop this change from this patch.